rpc/flipt/evaluation/evaluation.proto

```diff
@@ -84,3 +85,84 @@ service EvaluationService {
   rpc Variant(EvaluationRequest) returns (VariantEvaluationResponse) {}
   rpc Batch(BatchEvaluationRequest) returns (BatchEvaluationResponse) {}
 }
+
+message EvaluationDistribution {
+  string id = 1;
+  string rule_id = 2;
+  string variant_id = 3;
+  string variant_key = 4;
+  string variant_attachment = 5;
+  float rollout = 6;
+}
+
+message EvaluationRollout {
+  flipt.RolloutType type = 1;
+  int32 rank = 2;
+  oneof rule {
+    EvaluationRolloutSegment segment = 3;
+    EvaluationRolloutThreshold threshold = 4;
+  }
+}
+
+message EvaluationRolloutThreshold {
+  float percentage = 1;
+  bool value = 2;
+}
+
+message EvaluationRolloutSegment {
+  bool value = 1;
+  flipt.SegmentOperator segment_operator = 2;
```

Comment:
im wondering if it would be best to totally separate out this service from the existing `flipt.proto` messages and just re-create these messages here instead of importing them?

cc @GeorgeMac

Reply:
Hmm yeah you;re probably right. Having the hard coupling on these existing type could ultimately become restrictive.